insonmnia/oracle/oracle.go

```diff
@@ -118,36 +144,35 @@ func (o *Oracle) Serve(ctx context.Context) error {
 		zap.String("account", crypto.PubkeyToAddress(o.key.PublicKey).String()),
 		zap.String("price update period:", o.cfg.Oracle.PriceUpdatePeriod.String()),
 		zap.String("contract update period", o.cfg.Oracle.ContractUpdatePeriod.String()),
-		zap.Float64("deviation percent", o.cfg.Oracle.Percent))
+		zap.Float64("deviation percent", o.cfg.Oracle.Percent),
+		zap.Bool("from now", o.cfg.Oracle.FromNow))
```

Comment:
zap fields are represented as JSON, so please use underscore char for word splitting.